manager/integration/tests/test_node.py

```diff
@@ -1328,11 +1328,6 @@ def test_replica_datapath_cleanup(client):  # NOQA
 
     cleanup_host_disk(client, 'extra-disk')
 
-    # reset soft antiaffinity setting to false
-    node_soft_anti_affinity_setting = \
-        client.by_id_setting(SETTING_REPLICA_SOFT_ANTI_AFFINITY)
```

Comment:
We should also change the import to `common` instead of `test_scheduling` for all the references of `SETTING_REPLICA_SOFT_ANTI_AFFINITY`. We shouldn't define a constant twice.

Reply:
fixed